tools/cli/pkg/command/orchestration.go

```diff
@@ -134,6 +134,40 @@ Operations:
 {{- end }}
 `
 
+var kymaUpgradePreviewTpl = `Kyma Upgrade Preview
+Strategy:         {{.Parameters.Strategy.Type}}
```

Comment:
All the Strategy, Schedule, Workers and Kyma Version are optional parameters. Currently they don'T have meaningful default values, and server applies default values for them.
See https://github.com/berkanercan/control-plane/blob/main/tools/cli/pkg/command/upgrade.go#L45-L47

I suggest to introduce the following explicit default values in KCP CLI:
- strategy: parallel
- schedule: immediate
- Workers: 1